internal/cmd/api-key/command_delete.go

```diff
@@ -28,17 +26,45 @@ func (c *command) newDeleteCommand() *cobra.Command {
 
 func (c *command) delete(cmd *cobra.Command, args []string) error {
 	c.setKeyStoreIfNil()
-	apiKey := args[0]
 
-	promptMsg := fmt.Sprintf(errors.DeleteResourceConfirmYesNoMsg, resource.ApiKey, apiKey)
-	if ok, err := form.ConfirmDeletion(cmd, promptMsg, ""); err != nil || !ok {
+	if err := c.confirmDeletion(cmd, args); err != nil {
 		return err
 	}
-	httpResp, err := c.V2Client.DeleteApiKey(apiKey)
-	if err != nil {
-		return errors.CatchApiKeyForbiddenAccessError(err, deleteOperation, httpResp)
+
+	var errs error
+	var deleted []string
+	for _, id := range args {
+		if r, err := c.V2Client.DeleteApiKey(id); err != nil {
+			errs = errors.Join(errs, errors.CatchApiKeyForbiddenAccessError(err, deleteOperation, r))
```

Comment:
what does a joined error look like?

Reply:
+1, I don't see any examples in the golden files, so let's make sure we have at least one

Reply:
I only made one: `test/fixtures/output/context/delete/check-success-operation-fail.golden`

The format is:
```
Error: error_1
error_2
error_3
...
```

Reply:
I feel like I like the format of [multierror](https://github.com/hashicorp/go-multierror) better...

Reply:
I also like their format, but the default includes 2 extra newlines at the end that I found very annoying, so I used the built in `Join` instead.

But, looking at that repo, it looks like you can pass in a custom format. I think I'll switch to multierror and just write a format func that doesn't have the extraneous new lines.